specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingRequest.json

```diff
@@ -0,0 +1,504 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "pacebpexphot@microsoft.com"
+    },
+    "version": "2023-04-01"
+  },
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingRequests": {
```

Comment:
We need to expose list by BillingProfile, InvoiceSection and Customer or this is a PAS blocker